tests/kokkos/kokkos_utils_tests.cpp

```diff
@@ -288,9 +288,14 @@ void test_view_reduction(const Scalar a=Scalar(0.0), const int begin=0, const in
   Kokkos::View<Scalar*> results ("results", 1);
   const auto results_h = Kokkos::create_mirror_view(results);
 
+#ifdef KOKKOS_ENABLE_OPENMP
+  const int n_threads = omp_get_max_threads();
+#else
+  const int n_threads = 1;
```

Comment:
This always forces a team size of 1 on Cuda. You could ask for

```
team_size = ExeSpaceUtils<ExecSpace>::concurrency();
if (OnGpu<ExecSpace>::value) {
  team_size /= (is_single_precision<Real>::value ? 64 : 32);
}
```
That would give you all the available threads in OpenMP builds, and the number of warps in the block on Cuda.

Reply:
Can I simply just call `get_team_policy_force_team_size(1, VectorSize);`? Or do I need example with `packsize!=teamsize`?

The issue with the above is that `ExeSpace::concurrency() =  163,840` and I would get an error that team size should be `< 1024`. Another option is I could just set `team_size /= 161` on GPU.